src/components/BlogPage/BlogItem/BlogItem.spec.js

```diff
@@ -6,7 +6,7 @@ import BlogItem from './BlogItem';
 describe('<BlogItem />', () => {
   test('should match snapshot', () => {
     const wrapper = shallow(<BlogItem 
-      key='1'
+      key={Math.floor(Math.random() * 10)}
```

Comment:
why?

Reply:
This key is generated only once for the test purpose. This is done to avoid the warning in the console.

Reply:
Yeah, what was the problem?

Reply:
The following warning appears in the console: 
```
console.error node_modules/react/cjs/react.development.js:167
    Warning: Each child in a list should have a unique "key" prop.
    
    Check the top-level render call using <div>. See https://fb.me/react-warning-keys for more information.
        in BlogItem
```
I changed it back to '1'.